Readme.md

```diff
@@ -90,11 +90,27 @@ or if installed globally:
 legions
 ```
 
-## Functions Breakdown 
 
-|     Command     |   [Subcommand]    |                                  Description                                   |
+## Tests
+
+Install `pytest`:
+
+```bash
+pip install pytest
```

Comment:
It'd be nice to move the test dependencies to a separate requirements file. Usually you'd have `requirements_dev.txt` and do the installation with `pip install -r requirements_dev.txt`.
This has the advantage that when we later add docs, style checks, tox, etc. we can move these dependencies into a file and not rely on the readme. It's also easier to pin dependencies to versions that are known to work. :+1: